src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs

```diff
@@ -105,9 +114,10 @@ public string GetCurrentRequestHeader(ApiEvent eventInProgress)
 
             eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.ApiIdConstStrKey, out string apiId);
             eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.ForceRefreshId, out string forceRefresh);
+            eventInProgress.TryGetValue(MsalTelemetryBlobEventNames.RegionDiscovered, out string regionDiscovered);
 
             return $"{TelemetryConstants.HttpTelemetrySchemaVersion2}" +
-                $"|{apiId},{ConvertFromStringToBitwise(forceRefresh)}|";
+                $"|{apiId},{ConvertFromStringToBitwise(forceRefresh)}|{regionDiscovered}";
```

Comment:
>|{regionDiscovered} [](start = 69, length = 19)

nit: would have liked the platform config to be on a separate line, making it easier to see what is platform and what is general